cmd/zdb/zdb.c

```diff
@@ -7509,6 +7509,18 @@ mos_leak_log_spacemaps(spa_t *spa)
 		mos_obj_refd(sls->sls_sm_obj);
 }
 
+static void
+traverse_errorlog(objset_t *mos, uint64_t errlog)
```

Comment:
maybe rename this to something like `errorlog_count_refd`, similar to `bpobj_count_refd`?

Reply:
Sure!